### PR TITLE
Issues#322

### PR DESCRIPTION
**Pull request by nicolas-lefloch**
Issues #322

### CODE REVIEW

File: core/src/main/java/timeboard/core/model/VacationRequestStatus.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ * #L%
+ */
+
+public enum VacationRequestStatus {
+    PENDING("En attente"),
```
Comment (pierremellet) [inline core/src/main/java/timeboard/core/model/VacationRequestStatus.java]: Remove ui labels in enum.

Label must be provided by ui i18n system

File: core/src/main/java/timeboard/core/internal/events/VacationEvent.java [hunk 2]
```diff
@@ -30,12 +30,15 @@
...
 import java.util.Date;
 
+import static timeboard.core.internal.events.TimeboardEventType.*;
+
 
 public class VacationEvent extends TimeboardEvent {
     private VacationRequest request;
     private TimeboardEventType eventType;
 
     public VacationEvent(TimeboardEventType eventType, VacationRequest request) {
```
Comment (pierremellet) [inline core/src/main/java/timeboard/core/internal/events/VacationEvent.java]: Use object type or enum to define event type.

Don't use both

Comment (pierremellet) [reply core/src/main/java/timeboard/core/internal/events/VacationEvent.java]: Object type is more flexible.

It doest not need of a centralized enum to defined available event types.

File: core/src/main/java/timeboard/core/model/VacationRequestStatus.java (hunk 1 above)
Comment (nicolas-lefloch) [reply core/src/main/java/timeboard/core/model/VacationRequestStatus.java]: I use the same template in TaskStatus, maybe we should remove UI labels from both

File: core/src/main/java/timeboard/core/internal/events/VacationEvent.java (hunk 2 above)
Comment (nicolas-lefloch) [reply core/src/main/java/timeboard/core/internal/events/VacationEvent.java]: Here, the list of event types is exhaustive (CRUD).  No need of flexibility, enum seems to be the better solution.

File: core/src/main/java/timeboard/core/model/VacationRequestStatus.java (hunk 1 above)
Comment (pierremellet) [reply core/src/main/java/timeboard/core/model/VacationRequestStatus.java]: ok

File: core/src/main/java/timeboard/core/internal/events/VacationEvent.java (hunk 2 above)
Comment (pierremellet) [reply core/src/main/java/timeboard/core/internal/events/VacationEvent.java]: ok for the moment, need to invest if centralized enum won't be a SPOF

**Pull request merged**